### PR TITLE
Official Map Pool Overhaul for TD

### PR DESCRIPTION
**Pull request by dtluna**
It's time to do a map overhaul for Tiberium Dawn - it has stayed unchanged for way too long, and it is just horrific.

Most experienced players play the custom maps and the new players are stuck with the really bad maps. It's hurting our image.

Includes:

1. **Added 42 maps**
- 2p Antrax
- 2p Battlefield
- 2p Frontal Assault
- 2p Golden Nuggets (2P)
- 4p Braided River
- 4p Burning Hammer
- 4p Desert Storm III
- 4p Desert Storm V
- 4p Douze
- 4p Golden Nuggets (4P)
- 4p Guarana
- 4p Lost Forest
- 4p N/S Manufacturing Consent
- 4p Ping Pong Bay
- 4p Riverside Frontline
- 4p Search & Destroy
- 4p Sustain
- 4p Tiberian War (4P)
- 4p Too Hot
- 4p Valhalla
- 4p WarKraft
- 6p Alpha Juno V
- 6p Axis of Advance
- 6p Back to Basics (6P)
- 6p Botany
- 6p Control & Chaos
- 6p Golden Nuggets (6P)
- 6p Kolosseum
- 6p Sonatine
- 6p Tiberian War (6P)
- 8p Back to Basics (8P)
- 8p Control & Chaos
- 8p Golden Nuggets (8P)
- 8p Ground Zero
- 8p Order of Battle (8P)
- 8p Tiberium Connected
- 8p Twisted Peak
- 8p Vectors of Battle (8P)
- 10p Golden Nuggets (10P)
- 10p Order of Battle (10p)
- 10p Vectors of Battle (10P)
- 12p Golden Nuggets (12P)

2. **tweaked 10 maps** (author name, map name, tiberium trees)
- 2p Aggressive Tendencies
- 2p Avocado
- 2p Lost Souls
- 4p Escalations
- 4p Manufactoring Consent
- 4p Treasure Island
- 6p Haos Ridges
- 6p Vectors of Battle (6P)
- 8p Pressure
- 16p Morbid Aimless Poseidon

3. **removed 23 maps**
- 2p Chokepoint
- 2p Dead in Motion 2
- 2p Profit Over People
- 2p Sandstorm
- 2p Sea and Cake
- 2p The Hourglass
- 2p The Sentinel
- 2p Two Ponds
- 4p Break of Day
- 4p Deterring Democracy +
- 4p Hegemony or Survivor
- 4p Llamas
- 4p Minus Two
- 4p Mt.Nukebait
- 4p Necessary Illusions
- 4p Rogue States
- 4p Tiberium Oasis
- 4p White Acres
- 6p East vs West 3
- 6p Into the River Below
- 6p Llamas 2
- 8p Crossing the Rubicon
- 8p Hegemony or Survival 8P

[TD Map Pool thread on the forum.](http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19904)

### CODE REVIEW

**Comment by pchote** [comment]
TD only has 42 maps right now, so replacing 36 (86%!) of them in one release is going to be hugely disruptive for casual players who may not have installed extra maps.

I'll pull out a list of the least popular TD maps online, then we should only remove the 5-10 (max) worst ones now.  This can be repeated again over the next few releases as the stats evolve.

**Comment by dtluna** [comment]
@pchote Okay, we'll just delete them over time. I can wait :)
This PR is more about intention, what we should have in the end.

Although I think we should add all of the ones I propose: they are really popular.

BTW what's with the broken CI build?

**Comment by pchote** [comment]
> Although I think we should add all of the ones I propose: they are really popular.

The map stats should show this too.  Lets wait and see what they show, then adapt the change set accordingly.

> BTW what's with the broken CI build?

A couple of the maps that you have added fail our automated quality tests.  Click on the "details" link to see the errors.

**Comment by GraionDilach** [comment]
You shouldn't rename map filenames since these are also used for identification on the Resource Center. Also I am against memes in commit comments.

**Comment by pchote** [comment]
Online match counts for the shipped release-20161019 maps are:
```
1218 IN-IslandDuel.oramap
779 morbid-aimless-poseidon-2.oramap
601 deterring_democracy.oramap
474 hegemony_or_survival_8p.oramap
441 haos_ridges_cnc.oramap
425 deterring_democracy_plus.oramap
411 two-ponds.oramap
398 eastwest3.oramap
348 pressure_cnc.oramap
332 pirates_and_emperors.oramap
332 manufacturing_consent.oramap
306 garden2.oramap
288 Nullpeter.oramap
277 chokepoint.oramap
266 rubicon.oramap
255 llamas2.oramap
244 avocado.oramap
242 break_of_day.oramap
201 rogue_states.oramap
194 into_the_river_below.oramap
194 hegemony_or_survival.oramap
192 llamas.oramap
186 dead_in_motion_2.oramap
179 chord_simple.oramap
177 aggressivetendencies.oramap
163 the_hourglass.oramap
155 mtnukebait.oramap
145 lessons_from_kosovo.oramap
132 vectorsofbattle.oramap
119 thesentinel.oramap
118 white_acres.oramap
117 profit_over_people.oramap
114 treasure-island-2.oramap
102 Instant_Karma.oramap
84 escalations.oramap
82 minus_two.oramap
79 sandstorm.oramap
74 necessary_illusions.oramap
63 tiberium-oasis-cluster.oramap
50 sea_and_cake.oramap
42 the-hot-box
32 _lostsouls.oramap
```

Pulling the stats for custom maps is trickier, so I will do this later.

**Comment by pchote** [comment]
Another important point to consider is that we should keep a variety of maps, avoiding flooding the map roster with only symmetrical tournment-style maps (even if they are the most popular online).

**Comment by pchote** [comment]
Only three of the new maps show any online matches:
```
13 burning-hammer.oramap
29 twisted-peak.oramap
7 ground-zero.oramap
```

I assume that this is caused by something in this PR having changed the map hashes.  Can you please link to the resource center pages for each of your new maps?

**Comment by GraionDilach** [comment]
I don't like how this PR basically removes half of the most played 20 maps from the release.

**Comment by SoScared** [comment]
That's an overhaul all right.

As to removing popular maps remember this was done to RA as well simply because they were terrible maps and the fact that these were played a lot was a bad thing.

The one thing I've complained about a lot on TD the past few years is the map pool and I've made sure to hammer that point across to the TD community whenever I had the chance. It's awful, doesn't really invite to good team games, actually doesn't invite to games whatsoever and as mentioned in the OP hurts TD's image. For many years TD has had some of the best map makers in their community but has failed circulation the official map pool with their contributions.

I haven't been involved much in TD the past couple of years but these are very old observations and the fact that these are still relevant is a problem for TD. The TD map pool is just _so_ incredibly bad. It has been neglected release after release its literally starting to look like an antique collection. I'm certain the casual offline TD players would appreciate a serious flushing.

The TD map pool forum thread recently held off making an impact for the next release: http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=17430

Another observation if I may - as for the **80+** TD balance changes with the previous and the upcoming release, a lot of these has been play-tested on custom maps, not official maps, in order to balance the game properly. I think that needs some serious consideration. TD is being modified _with hindsight on custom maps_, the maps the active online community play and not the maps the mod advertises to the casual offline player base.

>The previous 2 release presented us with 37 balance changes for the TD mod and the current bleed is shipping 52 for the next release. Not saying these changes by themselves are bad but one could see how that might have isolated the competitive core and started putting some real distance between competitive and casual play. TD might need some time to settle before a new healthy rank pyramid grows back into the player base.
quote from: http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&p=299934#299934

This gap has been growing faster with each release and I don't see the gap closing by carefully add/removing official maps over the course of several releases. There's just no time and from my point of view, TD is running out of time making a future for itself.

IMO TD needs to take this risk and what better way to do it going hand-in-hand with the recent balance overhauls #12536 #11550 #11758. The potential benefits _far_ outweighs the risks and I'd wager a growing player base one year from now will thank you for it.

**Comment by SoScared** [comment]
> Also I am against memes in commit comments.

Funny you didn't seem to mind it in #12536

> AoAGeneral:AoATDGDNiceMeme

**Comment by pchote** [comment]
@SoScared makes a convincing point, so i'm willing to give hugely disruptive a chance.

I guess my only real concern is that these maps skew quite hard towards symmetric team maps.  I think it would be good if we can include some additional assymetrical/scenic/ffa maps to balance these out a bit.

**Comment by SoScared** [comment]
There are a few of the removed maps that are somewhat "iconic" such as 'Sea and Cake' (2p), 'The Hourglass' (2p) and 'Break of Day' (4p). These are pretty choky and cozy maps that would ease the transition. If there are new, better alternatives out there the TD community could easily swap them out later.

**Comment by dtluna** [comment]
@pchote

> I guess my only real concern is that these maps skew quite hard towards symmetric team maps. I think it would be good if we can include some additional assymetrical/scenic/ffa maps to balance these out a bit.

They do, because most of the experienced players like to play these maps, but as I've said I'm okay with not removing some of the maps that are already there. Which ones should remain, in your opinion?

**Comment by pchote** [comment]
I agree with @SoScared's picks.  Are there any other good new maps that we can include for these? (from the two map contests?)

**Comment by dtluna** [comment]
@pchote

> Are there any other good new maps that we can include for these? (from the two map contests?)

Those maps are awful though.

**Comment by SoScared** [comment]
Here's the results from the map contest of 2016: http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19525

**Comment by dtluna** [comment]
@pchote @SoScared 
I've brought **Sea and Cake** (2p), **The Hourglass** (2p), **Break of Day** (4p) and **Tiberium Oasis** (4p) back.

**Comment by GraionDilach** [comment]
The commit message is still a meme, @SoScared's reasoning is valid for the additions, but doesn't justify the most popular map removals - if the maps are that bad, then the next release onwards, their popularity should dwindle on the own. This PR as-is is feels too forced and just encourages my belief that TD is becoming more of a selected few's own mod promoted to official status instead of being a neutral OpenRA game as it's supposed to be.

Besides, East vs West 3 is the most playable 3v3 map in the mod.

No map called as Golden Nuggets exists on the Resource Center. Neither Frontal Assault. I haven't looked up the rest but I have a feeling there are more of such which also sounds like a problem.

**Comment by Micr0Bit** [comment]
>TD is becoming more of a selected few's own mod promoted to official status instead of being a neutral OpenRA game as it's supposed to be.
 
i feel the same for RA , "some" people think its necessary to replace every map with a strict balanced version and every spot and ore-patch should be mirrored ... additionally rules get approved by a group of people that play mostly 1v1's , giving the team-game aspect almost no attention

**Comment by SoScared** [comment]
@GraionDilach I'll gladly upload any official map that lacks in the resource center. I'm gonna have to go through the RA's official maps anyways as I suspect I might have deleted a map or two from the resource center.

Anyways, bad custom maps are played en masse across all the mods so I don't recognize the argument for this. This maps you mention are popular _only_ because they're in the official map pool. If they're weren't they'd be dead and gone years ago.

**Comment by dtluna** [comment]
@Micr0Bit

>additionally rules get approved by a group of people that play mostly 1v1's , giving the team-game aspect almost no attention

Most of the maps I have added are team maps. Most of the games that are being played in TD multiplayer are pretty big 3v3 or 4v4 maps. I don't know what you're talking about now here.

**Comment by Micr0Bit** [comment]
@dtluna : you dont need to feel accused , its just a general observation

>i feel the same for RA , "some" people think its necessary to replace every map with a strict balanced version of it (or even a new map) where every spot and ore-patch should be mirrored

^ "this" was more directed towards TD (the teamplay-aspect was more for RA alone , cause i dont have any idea about TD's balance)

**Comment by GraionDilach** [comment]
>Anyways, bad custom maps are played en masse across all the mods so I don't recognize the argument for this. These maps you mention are popular only because they're in the official map pool. If they're weren't they'd be dead and gone aeons ago.

Then I guess no harm is done if time proves your point by giving the old maps a chance to actually compete against the new ones and not being killed off under a Trump meme.

**Comment by SoScared** [comment]
And have 100+ maps in the official map pool? Don't see that happening. Regardless I think the above rationale speaks for itself.

**Comment by GraionDilach** [comment]
It lacks concrete proof however since there are no stats for 90% of the added maps. I don't buy that. Also 100+ maps shouldn't be a problem, popular mods like Twisted Insurrection or Mental Omega have map pools beyond 200 maps even.

**Comment by pchote** [comment]
This discussion is degrading into bad-spirited speculation from both sides, which is not at all helpful.  For the sake of having a productive discussion:

* I requested above links to these maps on the resource center so that we can look at stats as well as having a papertrail on where these came from.  This still stands.
* @GraionDilach: Are there any specific maps that you particularly object to removing ("all of them" is not going to be helpful).  If you have a good argument for keeping them then @dtluna has already said that he's happy to keep them.

Personally, I don't like the idea of having super-large (more than say 90 - 100) official map pools in any of our mods.  That just smells of poor curation.

**Comment by GraionDilach** [comment]
EvW3 is the sole map I play TD on.

**Comment by dtluna** [comment]
@pchote 
I have removed all the custom rules.

- 2p [Antrax](http://resource.openra.net/maps/16796/)
- 2p [Battlefield](http://resource.openra.net/maps/16958/)
- 2p [Frontal Assault](http://resource.openra.net/maps/16131/)
- 2p [Golden Nuggets (2P)](http://resource.openra.net/maps/18130/)
- 4p Braided River - deleted from the website by InsertName
- 4p [Burning Hammer](http://resource.openra.net/maps/17176/)
- 4p [Desert Storm III](http://resource.openra.net/maps/15079/)
- 4p [Desert Storm V](http://resource.openra.net/maps/15344/)
- 4p [Douze](http://resource.openra.net/maps/18009/)
- 4p [Golden Nuggets (4P)](http://resource.openra.net/maps/18129/)
- 4p [Guarana](http://resource.openra.net/maps/16922/)
- 4p [Lost Forest](http://resource.openra.net/maps/17126/)
- 4p [N/S Manufacturing Consent](http://resource.openra.net/maps/16938/)
- 4p [Ping Pong Bay](http://resource.openra.net/maps/13534/)
- 4p [Riverside Frontline](http://resource.openra.net/maps/12844/)
- 4p [Search & Destroy](http://resource.openra.net/maps/17143/)
- 4p [Sustain](http://resource.openra.net/maps/16963/)
- 4p [Tiberian War (4P)](http://resource.openra.net/maps/16970/)
- 4p [Too Hot](http://resource.openra.net/maps/11902/)
- 4p [Valhalla](http://resource.openra.net/maps/16709/)
- 4p [WarKraft](http://resource.openra.net/maps/15334/)
- 6p [Alpha Juno V](http://resource.openra.net/maps/13528/)
- 6p [Axis of Advance](http://resource.openra.net/maps/15565/)
- 6p [Back to Basics (6P)](http://resource.openra.net/maps/13423/)
- 6p [Botany](http://resource.openra.net/maps/15698/)
- 6p [Control & Chaos](http://resource.openra.net/maps/13265/)
- 6p [Golden Nuggets (6P)](http://resource.openra.net/maps/18093/)
- 6p [Kolosseum](http://resource.openra.net/maps/16934/)
- 6p [Sonatine](http://resource.openra.net/maps/17566/)
- 6p [Tiberian War (6P)](http://resource.openra.net/maps/18081/)
- 8p [Back to Basics (8P)](http://resource.openra.net/maps/18259/)
- 8p [Control & Chaos](http://resource.openra.net/maps/17327/)
- 8p [Golden Nuggets (8P)](http://resource.openra.net/maps/15647/)
- 8p [Ground Zero](http://resource.openra.net/maps/18029/)
- 8p [Order of Battle (8P)](http://resource.openra.net/maps/18037/)
- 8p [Tiberium Connected](http://resource.openra.net/maps/17445/)
- 8p [Twisted Peak](http://resource.openra.net/maps/17184/)
- 8p [Vectors of Battle (8P)](http://resource.openra.net/maps/17324/)
- 10p [Golden Nuggets (10P)](http://resource.openra.net/maps/17367/)
- 10p [Order of Battle (10p)](http://resource.openra.net/maps/18090/)
- 10p [Vectors of Battle (10P)](http://resource.openra.net/maps/18323/)
- 12p [Golden Nuggets (12P)](http://resource.openra.net/maps/17531/)

**Comment by Phrohdoh** [comment]
Can we get a commit message that will be helpful when reviewing history?
No one is going to grep for a political reference.

**Comment by dtluna** [comment]
@Phrohdoh I'll change it when we come up with a final map lineup.

**Comment by AoAGeneral** [comment]
This map has the fixed spawn locations. The link in the Map Pool is invalid because it got deleted from the center. http://resource.openra.net/maps/19523/

So this one can be updated over the official version.

**Comment by dtluna** [comment]
@AoAGeneral Sure, I'll add it.

**Comment by AoAGeneral** [comment]
What do you think of replacing Deterring Democracy with Deterring Democracy revised? http://resource.openra.net/maps/12745/

The revised version cleans up the tiberium and places the players in the center in vs west to east. There is also the removal of the blue field/tree from either side so they only have one to expand to in their area instead of two.

A bit relunctant despite it being a good map since the original has a bunch of plays compared to this one. On the resource itself it has about 410 downloads.

**Comment by GraionDilach** [comment]
Well, since I have repeated myself two times already, I will definitely oppose this PR if it drops East vs West 3 because I don't see why that is a bad map.

**Comment by dtluna** [comment]
@GraionDilach
Could you please describe the typical gameplay on East vs West 3? Do you think it favors one faction over another?

**Comment by GraionDilach** [comment]
Lemme guess, it favors GDI over Nod because it's a wide map with both teams in a single line and so players can monitor their opponent's cargo planes, and that's BAD BAD BAD.

You know what, forget what I said. Carry on, even keep the Trump meme commit text because Map Pool is Great&trade; now.

**Comment by AoAGeneral** [comment]
Actually no the map may favor Nod. Because of the open lanes bikes/buggy plays can go very well on this map. Even more so with the Obelisk fix it may become interesting with how devestating these defense structures can be. I think GDI would actually be weaker on this map.

But im curious on East vs West 3 map itself since I have seen some games pop up on it vs bot games. Going by the ORA resource center it has about 1 play times on it (Older versions) and 79 views with 19 downloads. With the older version added its about 150+ views on it with 60+ downloads. (This of course doesn't count offline plays).

I remember I used to play on this map on and off in games but this was a long time ago. I think its turned into a fun bot map as other mappers have jumped into the scene. (Casual players in this sense able to hold up and defend waves of attackers without much needing expansions or fun 1v3 bot AI challenges for casual players.)

**Comment by reaperrr** [comment]
In my opinion EvW3 should stay. The bio lab + guard towers in the middle are a nice twist and it's one of the better small maps for games vs. (multiple) bots. Skirmish/AI testing shouldn't be completely ignored when assembling the map pool.

Keeping just one more map around shouldn't do any harm.

**Comment by dtluna** [comment]
@AoAGeneral You actually have a point here. I'll include it back.

**Comment by SoScared** [comment]
Weighing in as requested.

Appears as a map pool has been compromised between the participants.

Visually the map pool looks way more appealing, team-game friendly and will surely encourage more map making within the TD community for the future.

Also a reminder to @dtluna to rename the maps 'Golden Nuggets' vs 'Valley of Gold' for the group of 6 maps. Agreed on 'Valley of Gold' being a way more catchy map name.

As mentioned above I'll gladly take responsibility for official maps being present in the resource center in conjuction with https://github.com/OpenRA/OpenRA-Resources/issues/295.

:+1:

**Comment by dtluna** [comment]
Cleaned up some tiles on "Valley of Gold" and the spawn letters.

I think I am done.

@pchote what do you think?

**Comment by pchote** [comment]
For completeness here are the stats for the new maps.  First column is the map_stats play count (same as for the official maps above), second column is the number of times it has been downloaded from the resource center, third column is the map name.
```
306 1845 "Control and Chaos"
232 1130 "Valley of Gold"
201 399 "AlphaJunoIV"
143 887 "Ping Pong Bay"
74 806 "Back to Basics"
69 627 "Too Hot"
52 427 "desert storm III"
47 156 "Fronteinsatz"
32 336 "Tiberian War"
29 367 "Twisted Peak"
25 448 "Black Valley of Gold"
25 185 "sustain"
23 341 "Axis of Advance"
23 133 "=antrax="
21 372 "Riverside Frontline 02"
21 334 "Botany"
21 324 "WarKraft"
21 230 "Valley of Goldnugget"
21 197 "Guarana"
20 472 "desert storm V"
16 330 "Control and Chaos 8pl"
16 232 "Kolosseum"
15 263 "Black Vectors of Battle"
13 168 "Burning Hammer"
12 268 "Black Valley of Gold VI"
12 145 "Valley of Goldnugget 2v2"
11 162 "m. consent clone ns"
11 136 "lostforest"
9 43 "Valley of Goldnugget 1v1"
7 152 "Ground Zero"
7 149 "douze"
7 112 "Sonatine"
5 99 "Tiberium Connected 8pl"
4 166 "Desert Vectors of Battle V"
3 68 "BLACK Tiberian War"
3 168 "Black Order of Battle"
2 91 "Back to Basics 4v4"
2 25 "Order of Battle"
1 47 "=schlachtfeld="
1 44 "Walhalla"
1 43 "Koordinieren und Ausradieren"
```

**Review by pchote — CHANGES_REQUESTED** [review]
I've finally managed to find time to look at these in more detail.  Some of the additions look like great maps, and some of the removals are *really* overdue.  There are only a small number of changes I disagree with:

* Please add back Tiberian Garden and Chokepoint.  I use these for testing the AI, and they're among the most popular shipped maps.

* I don't like the way that Sonatine, Douze, Search & Destroy, wall off tib/tech in such an obviously constructed way.  It feels like a natural part of the map design in Kolosseum, so I don't have a problem with it there.

* Valhalla, Battlefield, and Lost Forest feel badly undetailed/unpolished.  Frontal assault, WarKraft, Back to Basics, Tiberium Connected could do with more detail, but aren't bad enough to reject.  There really is no excuse to have 99.9% of the map as the default grass/sand type, cliff, or tree.

* It feels a bit unpolished and/or unprofessional to use tiberium to draw out symbols on the minimap in Valley of Gold and Orders of Battle.  They look reasonable otherwise, so i'm not asking to remove them... but it would be nice if future revisions could fix this.

* I see "Valley of Gold", "Valley-of-Gold", and even "Valley go Gold".  Please unify these names! (also, do we really need to ship 6 variants of the same map?)

* Tiberian War and Alpha Juno V look a bit chokey.  How well do they perform in MP?

In summary I'd like to see:

* Tiberian Garden and Chokepoint added back.
* Sonatine, Douze, Search & Destroy, Valhalla, Battlefield, and Lost Forest removed.
* "Valley-of-Gold" etc renamed to "Valley of Gold".

:+1: after those fixes.

**Comment by dtluna** [comment]
> Please add back Tiberian Garden and Chokepoint. I use these for testing the AI

That's actually pretty bad. These maps are not the types of maps that are being played in the multiplayer. Maybe that's why AI cannot do anything on a more open map?

>Tiberian War and Alpha Juno V look a bit chokey. How well do they perform in MP?

They actually don't feel choky at all. They have enough space to maneuver and do sneaky tactics.

>I don't like the way that Sonatine, Douze, Search & Destroy, wall off tib/tech in such an obviously constructed way.
>Valhalla, Battlefield, and Lost Forest feel badly undetailed/unpolished.
>It feels a bit unpolished and/or unprofessional to use tiberium to draw out symbols on the minimap in Valley of Gold and Orders of Battle.

I believe the first thing we should care is gameplay, not "professionalism" or aesthetics. It's good when map looks good, but it's nothing when the gameplay is just stalemate.

Nevertheless, I'll make the changes you request.

**Comment by pchote** [comment]
> I believe the first thing we should care is gameplay, not "professionalism" or aesthetics. It's good when map looks good, but it's nothing when the gameplay is just stalemate.

I'm not saying that gameplay shouldn't come first, but that gameplay by itself is not sufficient.  Our choice of shipped maps are supposed to reflect the best we have to offer, and so we really should be requiring *both*.

**Comment by reaperrr** [comment]
> I'm not saying that gameplay shouldn't come first, but that gameplay by itself is not sufficient. Our choice of shipped maps are supposed to reflect the best we have to offer, and so we really should be requiring both.

Agreed.

**Comment by Micr0Bit** [comment]
> I believe the first thing we should care is gameplay, not "professionalism" or aesthetics. It's good when map looks good, but it's nothing when the gameplay is just stalemate.

One of the most played 1v1 maps in RA is OreLord (also in leagues/tournys) ... has been under investigation several times and claimed to be "unbalanced" from the self-announced balance-pros ... 
One of the reasons why its so popular ,is because it is in the official map-pool and easier/faster to access (on top of the map-pool and a 1v1 map)

And i think it is because its visually appealing ... , especially for strangers , that can have more value on what map they choose to play

A new OpenRA-player (+friends) most likely has no idea about the balance ... and/or doesnt even care ... so he will pick a map for specific visual  details like for example size (count of the spots) , maybe maps where all the spots are in the corners so they  have better protection ... lots of ore ... islands ... and so on ... and i think "maps that are visually appealing" because of how they are designed ... certain ore-shapes are just enjoyable to see (harvest?) ... or little details that make a map unique and different from others 

let alone the maps that get played over and over again by competitive players and therefor eventually have more weight on the decision "what" goes into the map-pool ... ITS NOT all about balance 

maybe theres people that just want to have fun on a lan-party ... with a couple of friends ... or even two brothers door on door ... those people dont give a *f* on how perfectly mirrored maps are

**Comment by dtluna** [comment]
>Our choice of shipped maps are supposed to reflect the best we have to offer

Then why are Chokepoint, Tiberium Garden and East vs West in the map pool?

**Comment by pchote** [comment]
That was already addressed in the discussion above.

The playtest feature freeze begins tomorrow (planned and warned for over a week now).  Please update this ASAP if you want to get these map changes merged in time.

**Comment by dtluna** [comment]
@pchote Done, please check it out.

**Review by pchote — APPROVED** [review]

**Comment by reaperrr** [comment]
I'd +1 this if it wasn't for one inconsistency: Morbid Aimless Poseidon is listed as 'modified' in the OP, but this PR actually removes it. Considering it's the 2nd-most popular map according to the stats @pchote posted, and the only 16 player map, I'd say that's a merge blocker, as the discussion doesn't indicate any reasons why it would be removed.

I'm also somewhat concerned that the total map count only goes up by 15 even though going by the lists in the OP +/- the changes that were requested, we should be at +18 (6 of 23 removed maps were added back, 6 of 42 added maps were requested to be removed). One of the 3 'missing' maps is MAP, obviously, but I wonder whether there might be 1 or 2 more silent removals.

Anyway, the overall map pool looks good to me otherwise, so if you add your modified Morbid Aimless Poseidon back, this gets my approval.

**Comment by dtluna** [comment]
@reaperrr 
I apologize for errors in the list.

> but I wonder whether there might be 1 or 2 more silent removals.

They're not silent after all, because you can look into the changes list.

**Comment by reaperrr** [comment]
@pchote Merging this is fine by me now, but the list in the OP is outdated and right now I don't have time to manually go through the commit to write a detailed changelog entry. Would a single changelog entry pointing to this PR be enough (we can update the list in the OP later)?

**Comment by pchote** [comment]
Fine by me.  We can update it between the playtests the final release.

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/e75220e6cf10cf8c19032b76cd346930bd24af1c...928ebd9f96adebf5e6c3c598499862ce4335f6d6)

**Comment by Irnub** [comment]
Hi all,

Re all these maps:

-Axis of Advance updated, less tib than in the playtest version and just now updated again to give the tib a more naturalistic appearance:

http://resource.openra.net/maps/19823/

-Re comments above, Valley of Gold updated to remove the tib patterns and give a more naturalistic appearance:

http://resource.openra.net/maps/19825/

-Re above comments again, Order of Battle updated to remove the tib patterns and give a more naturalistic appearance:

http://resource.openra.net/maps/19824/

Lastly, I've also noticed that the 10p desert version of Vectors of Battle included in the playtest is credited to Black when it should be credited to pulse/Black

:)
